Cargo.toml

```diff
@@ -10,8 +10,7 @@ categories = ["development-tools"]
 build = "build.rs"
 
 [dependencies]
-# cargo hash delivered with nightly-2018-03-14
-cargo = { git = "https://github.com/rust-lang/cargo", rev = "5f83bb4044f32b60d06717c609610f67411fc671" }
+cargo = { git = "https://github.com/rust-lang/cargo" }
```

Comment:
Could we keep the `rev`? I think it is useful for RLS devs to have a consistent version of Cargo, even if when we embed in Rust, we get whatever Rust has.

Reply:
Sure yeah, but just to confirm, doesn't `Cargo.lock` do that already? With the `rev` in here it means that Cargo can't be updated w/o editing the `Cargo.toml`

Reply:
Referring to the hash allows frequent use of `cargo update` in a non-breaking way. We had it in the lock before but that seemed to encourage not using `cargo update`. See #680. Maybe there's a better way?

Reply:
Ok! For big projects I wouldn't necessarily recommend `cargo update` as it can change quite a lot but I'll add this back in